build.py

```
@@ -40,7 +41,7 @@ def get_deb_extra_depends() -> str:
     return ""
 
 def system2(cmd):
-    exit_code = os.system(cmd)
+    exit_code = subprocess.run(cmd, shell=True).returncode
```

Ruff (0.16.0)

[error] `subprocess` call with `shell=True` identified, security issue

(S602)